controllers/admin/AdminGroupsController.php

```diff
@@ -633,7 +633,11 @@ public function displayEditLink($token, $id)
         $href = self::$currentIndex . '&' . $this->identifier . '=' . $id . '&update' . $this->table . '&token=' . ($token != null ? $token : $this->token);
 
         if ($this->display == 'view') {
-            $href = Context::getContext()->link->getAdminLink('AdminCustomers') . '&id_customer=' . (int) $id . '&updatecustomer&back=' . urlencode($href);
```

Comment:
here's another example, even though url is generated, but parameters are hardcoded.

Reply:
LegacyUrlConverter should work here 🤔 if it does not, we need to improve it

Reply:
This is the worst case scenario with url conversions ^^ With pure hard coded urls they can be caught by the `LegacyUrlListener` and are redirected to symfony urls, but in this case the `getAdminLink` will return the symfony index page..

Reply:
OK @jolelievre told me this one won't do because of this mix :/ but I cant see a solution here. We can only upgrade prestashop and pray that overrides or modules did not do this.

Reply:
Sadly some did.. even in PrestaShop modules this kind of link exist (you can't blame them it is used in the core...) That's why we added a warning in the documentation so that people should be careful with the way they generate their link https://devdocs.prestashop.com/1.7/development/architecture/migration-guide/controller-routing/#the-legacy-link-auto-redirection